site/en/docs/extensions/reference/tabs/index.md

```diff
@@ -1,139 +1,255 @@
 ---
 api: tabs
-extra_permissions_html:
-  The majority of the <code>chrome.tabs</code> API can be used without declaring any permission. However, the <code>"tabs"</code> permission is required in order to populate the <code>url</code>, <code>pendingUrl</code>, <code>title</code>, and <code>favIconUrl</code> properties of <code><a href="#type-Tab">Tab</a></code>.
 ---
 
-## Manifest
-
-You can use most `chrome.tabs` methods and events without declaring any permissions in the
-extension's [manifest][manifest] file. However, if you require access to the `url`, `pendingUrl`,
-`title`, or `favIconUrl` properties of [`tabs.Tab`][tab], you must declare the `"tabs"` permission
-in the manifest, as shown below:
-
-```json
-{
-  "name": "My extension",
-  ...
-  "permissions": [
-    "tabs"
-  ],
-  ...
-}
-```
+## Overview
 
-## Examples
+The Tabs API not only offers features for manipulating and managing tabs, but can also detect the
+[language][tabs-detect-language] of the tab, take a [screenshot][tabs-capture], and
+[communicate][tabs-message] with a content script. 
 
-The following sections demonstrate several common use cases for the chrome.tabs API.
+{% Aside %}
 
-### Opening an extension page in a new tab
+The Tabs API can be used in the service worker and extension pages, but not in content scripts.
 
-A common pattern for extensions is to open an onboarding page in a new tab when the extension is
-installed. The following example shows how to do this.
+{% endAside %}
+
+## Manifest {: #manifest }
+
+Most functions don't require any permission, like, for example: [creating][tabs-create] a new tab,
+[reloading][tabs-reload] a tab, [navigating][tabs-update] to another URL, etc.
+
+### Permissions {: #perms }
+
+A permission is only required to access the `url`, `pendingUrl`, `title`, and `favIconUrl` of
```

Comment:
I think it may be helpful to group these 4 properties together and refer to them collectively as "sensitive Tab properties" or something along those lines. This would give us a shorthand way of referring to them collectively and to discuss them in other contexts. For example, we would be able to say something like "host permissions grant an extension the ability to interact with sensitive properties on matching tabs."

Reply:
Good idea, I revised this sentence.